yogstation.dme

```diff
@@ -1467,6 +1467,7 @@
 #include "code\modules\admin\view_variables\topic_basic.dm"
 #include "code\modules\admin\view_variables\topic_list.dm"
 #include "code\modules\admin\view_variables\view_variables.dm"
+#include "code\modules\antagonists\sentient_creature.dm"
```

Comment:
?

Reply:
I ddont know why that has been added. The only code i added was for the drinks? Should I just remove it?

Reply:
it's usually best if only lines pertaining to the PR are touched

Reply:
All good, I don't know why this happened since the Syndicate Bomb was the only code I did touch. But it has been fixed now.